packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -132,15 +122,18 @@ const config = ${JSON.stringify(config, null, 2)}
 ${buildDirname(edge, relativeOutdir)}
 ${buildRuntimeDataModel(this.dmmf.datamodel)}
 
-${await buildInlineSchema(dataProxy, schemaPath)}
-${buildInlineDatasource(dataProxy, datasources)}
 ${buildInjectableEdgeEnv(edge, datasources)}
 ${buildWarnEnvConflicts(edge, runtimeDir, runtimeName)}
 ${buildDebugInitialization(edge)}
 const PrismaClient = getPrismaClient(config)
 exports.PrismaClient = PrismaClient
 Object.assign(exports, Prisma)${deno ? '\nexport { exports as default, Prisma, PrismaClient }' : ''}
-${buildNFTAnnotations(dataProxy, engineType, platforms, relativeOutdir)}
+${buildNFTAnnotations(
+  edge || false /** TODO after removal of dataProxy, do if --no-engine or edge */,
+  engineType,
+  platforms,
+  relativeOutdir,
+)}
```

Comment:
We don't build annotations for edge runtime builds, and in a follow up PR we will also introduce `--no-engine` which will lock a user to use Data Proxy.